workspace.go

```diff
@@ -65,6 +67,15 @@ type Workspaces interface {
 
 	// UnassignSSHKey from a workspace.
 	UnassignSSHKey(ctx context.Context, workspaceID string) (*Workspace, error)
+
+	// Add remote state consumers to a workspace.
+	AddRemoteStateConsumers(ctx context.Context, workspaceID string, options WorkspaceAddRemoteStateConsumersOptions) error
+
+	// Remove remote state consumers from a workspace.
+	RemoveRemoteStateConsumers(ctx context.Context, workspaceID string, options WorkspaceRemoveRemoteStateConsumersOptions) error
+
+	// ReadRemoteStateConsumers reads the remote state consumers for a workspace.
+	ReadRemoteStateConsumers(ctx context.Context, workspaceID string) (*WorkspaceList, error)
 }
```

Comment:
You can `PATCH` remote state consumers, too; Might want to add that in, as for users with massive amounts of workspace may find it useful if they're doing their own bespoke API setup (even if our provider just always uses add/remove)

Reply:
added